### DISCUSSION TITLE
ÖYG 1 Aralık Ödevleri

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#450] [Bilsem]
2 den 1000 e kadar asal sayıları yazdıran (aralarında virgülle ) programı yazınız.

Bilgisayarın rastgele tuttuğu bir sayıyı kullanıcının bulmaya çalıştığı programı yazınız. (Bilgisayar yukarı , aşağı şeklinde yönlendirme yapacaktır)
Programı aşağıdaki koddan sonra yazmaya devam ettiriniz.

```python
import random

bts = random.randint(1,100) #1 le 100 arasında rastgele sayı tutuluyor ve bts değişkenine atanıyor
```

Kullanıcıya bir sayı sorun. Girilen sayı kadar * lardan üçgen yapın. Örneğin 3 girildiğinde:

```
  *
 * *
* * *
```
şeklinde.

Ödevlerinizi bilisimbilsem@gmail.com adresine py formatında dosyalarla gönderiniz.

**Comment by eymensarac** [comment]
`def asal_sayilari_yazdir():
    asal_sayilar = []
    for num in range(2, 1001):
        for i in range(2, int(num**0.5) + 1):
            if num % i == 0:
                break
        else:
            asal_sayilar.append(num)
    print('Asal Sayılar: ', ', '.join(map(str, asal_sayilar)))

asal_sayilari_yazdir()

# Kullanıcıdan sayı alıp üçgen yapmak için
n = int(input("Kaç *'dan üçgen yapmak istersiniz? "))
for i in range(1, n + 1):
    print(' ' * (n - i) + '* ' * i)
`

**Comment by eymensarac** [comment]
```
def asal_sayilari_yazdir():
    asal_sayilar = []
    for num in range(2, 1001):
        for i in range(2, int(num**0.5) + 1):
            if num % i == 0:
                break
        else:
            asal_sayilar.append(num)
    print('Asal Sayılar: ', ', '.join(map(str, asal_sayilar)))

asal_sayilari_yazdir()

# Kullanıcıdan sayı alıp üçgen yapmak için
n = int(input("Kaç *'dan üçgen yapmak istersiniz? "))
for i in range(1, n + 1):
    print(' ' * (n - i) + '* ' * i)
```
